docs/content/guides/author-apps/kubernetes/how-to-access-secrets/index.md

```diff
@@ -0,0 +1,132 @@
+---
+type: docs
+title: "How-To: Access Kubernetes secrets using PodSpec"
+linkTitle: "Secrets using PodSpec"
+description: "Learn how to patch Kubernetes secrets into the container environment using PodSpec definitions"
+weight: 300
+slug: 'secrets-podspec'
+categories: "How-To"
+tags: ["containers","Kubernetes", "secrets"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Patch existing Kubernetes secrets using [PodSpec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) definitions and provide them to the environment of a container.
+
+## Prerequisites
+
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Step 1: Define a container
+Begin by creating a file named `app.bicep` with a Radius [container]({{< ref "guides/author-apps/containers" >}}):
+
+{{< rad file="snippets/secrets-container.bicep" embed=true >}}
+
+## Step 2: Deploy the app and container
+```bash
+rad run ./app.bicep -a demo
+```
+
+Once the deployment completes successfully, you should see the following confirmation message along with some system logs:
+
+```bash
```

Comment:
for depicting console output, you can remove the `bash` tag in the code block

Reply:
Updated to incorporate